foter done

kait-meshcheriakova opened the pull request.

kait-meshcheriakova committed:
foter done

daniil0040 reviewed (changes requested):
Пофіксити
1.Не працюють SVG
2.Посилання на оті файли що ніхто не читає мають бути напів прозорі
3.Застилізувати слоган,зробити так щоб його було краще видно
4.Відділити від основного контенту на мобайл та таблет версіях

kait-meshcheriakova committed:
footer

мейн оновлений але саме в цій гілці оновленого не бачу. прийми пул я оновлення підтягну і шляхи для свг пропишу

kait-meshcheriakova committed:
Merge branch 'main' into footer

kait-meshcheriakova committed:
foter svg
